gluten-core/src/main/scala/io/glutenproject/execution/BasicScanExecTransformer.scala

```diff
@@ -85,18 +85,19 @@ trait BasicScanExecTransformer extends TransformSupport {
   }
 
   override def doValidate(): Boolean = {
-    // TODO need also check orc file format and also move this check in native.
-    ConverterUtils.getFileFormat(this) match {
+    val fileFormat = ConverterUtils.getFileFormat(this)
+    if (!BackendsApiManager.getTransformerApiInstance.supportsReadFileFormat(fileFormat)) {
+      logDebug(
+        s"Validation failed for ${this.getClass.toString} due to $fileFormat is not supported.")
+      return false
+    }
+    fileFormat match {
       case ParquetReadFormat =>
         if (BackendsApiManager.getBackendName.equals("velox") &&
           unsupportedDataType()) {
           return false
         }
-      case DwrfReadFormat =>
       case _ =>
-        if (BackendsApiManager.getBackendName.equals("velox")) {
-          return false
-      }
```

Comment:
Replaced by validating file format according to the setting of each backend.

Reply:
Can we somehow remove `BackendsApiManager.getBackendName.equals("velox")` from `case ParquetReadFormat` too?

Reply:
I guess it's here to prevent some unsupported types. How about moving it to the native validation logic?

Reply:
I think we can move 
```
2. fileFormat match {
case ParquetReadFormat =>
if (BackendsApiManager.getBackendName.equals("velox")
```
to supportsReadFileFormat

Reply:
@zhztheplayer @jinchengchenghh Moved the type validation to `supportsReadFileFormat`. Would you like to review again?